system/jlib/jlib.hpp

```diff
@@ -275,8 +275,6 @@ enum DAFSConnectCfg { SSLNone = 0, SSLOnly, SSLFirst, UnsecureFirst };
 
 #include "jstring.hpp"
 #include "jarray.hpp"
-#include "jhash.hpp"
-#include "jstream.hpp"
```

Comment:
NB: there were removed to resolve a circular dependency issue.
jlib.hpp is included very widely. There were a couple of places I had to add #include "jhash.hpp" after removing these (see top of this PR).

Reply:
What was the circular dependency?  That suggests something needs cleaning up.  jlib.hpp was meant to be include everything from jlib (possibly not a great idea).